build/FrameworkPackage/MakeFrameworkPackage.ps1

```diff
@@ -74,40 +65,35 @@ ForEach ($input in ($inputs -split ";"))
     Write-Verbose "Copying $inputBasePath\Themes"
     Copy-IntoNewDirectory -IfExists $inputBasePath\Themes $fullOutputPath\PackageContents\Microsoft.UI.Xaml
 
+    $sdkReferencesPath=$kitsRoot10+"\References\10.0.18323.0";
+    $wimdReferences = $sdkReferencesPath + "\Windows.Foundation.FoundationContract\3.0.0.0\Windows.Foundation.FoundationContract.winmd;" + $sdkReferencesPath + "\Windows.Foundation.UniversalApiContract\8.0.0.0\Windows.Foundation.UniversalApiContract.winmd"
```

Comment:
>3 [](start = 83, length = 1)

These version numbers change each release, we should probably use a "gci -r Windows.Foundation.FoundationContract.winmd" to find the thing and be resilient to that changing.